openupgrade_scripts/scripts/account/18.0.1.3/pre-migration.py

```diff
@@ -169,6 +169,25 @@ def fill_account_payment(env):
     )
 
 
+def fill_statement_line_fields(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_bank_statement_line sl
+        SET company_id = mv.company_id
+        FROM account_move mv
+        WHERE sl.move_id = mv.id""",
```

Comment:
```suggestion
        WHERE sl.move_id = mv.id
        AND sl.company_id IS NULL
        """,
```

Reply:
Is there any reason company_id might not be NULL ? Field was not stored in 17.0.
Or just a good practice ?

Reply:
Oh, forget it. You are creating the columns upper, so no need.